packages/private/demo/__stories__/math.tsx

```diff
@@ -94,3 +85,24 @@ storiesOf('Math', module)
       </div>
     )
   })
+  .add('MathEditor (Overlay does not hide previous text)', () => {
+    return (
+      <RootThemeProvider
+        theme={{ editor: defaultEditorTheme, renderer: defaultRendererTheme }}
+      >
+        <p>
+          Consequatur quos vitae vel in. Suscipit a eius deserunt nisi neque.
+          Minima dolores quia quo veritatis. Aut natus nihil necessitatibus
+          delectus ea eligendi dolor sit. Neque deleniti vel sint laborum et
+          quia.
+        </p>
+
+        <p>
+          Voluptatem similique perspiciatis earum eius fugit officiis ut et.
+          Numquam laudantium ut ipsum. Fugiat voluptatum dolores sit maiores.
+        </p>
+
+        <MathEditorExample state="\sqrt{2}" visual />
+      </RootThemeProvider>
+    )
+  })
```

Comment:
Could you please revert the changes in this file? I don't think we need that additional story in the long-term.